inc/class-sitemaps.php

```diff
@@ -124,7 +124,7 @@ public function reduce_query_load() {
 		$request_uri = $_SERVER['REQUEST_URI'];
 		$extension   = substr( $request_uri, -4 );
 
-		if ( false !== stripos( $request_uri, 'sitemap' ) && ( in_array( $extension, array( '.xml', '.xsl', ) ) ) ) {
+		if ( false !== stripos( $request_uri, 'sitemap' ) && ( in_array( $extension, array( '.xml', '.xsl' ) ) ) ) {
```

Comment:
It's the new rule for single line arrays. So trailing coma is enforced for multi-line arrays (eases moving lines, adding lines), but un-enforced for single line.